rotors_gazebo_plugins/CMakeLists.txt

```diff
@@ -23,8 +23,8 @@ project(rotors_gazebo_plugins)
 # =============================================================================================== #
 
 if(NOT DEFINED BUILD_MAVLINK_INTERFACE_PLUGIN)
-  message(STATUS "BUILD_MAVLINK_INTERFACE_PLUGIN variable not provided, setting to FALSE.")
-  set(BUILD_MAVLINK_INTERFACE_PLUGIN FALSE)
+  message(STATUS "BUILD_MAVLINK_INTERFACE_PLUGIN variable not provided, but anyway setting to TRUE.")
```

Comment:
This doesn't make sense to change here, we can set the variable to TRUE by default but not like this ;) (why do you want it to be TRUE by default?)